docs/outlook/append-on-send.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Implement append on send in your Outlook add-in (preview)
+description: Learn how to implement the append-on-send feature in your Outlook add-in.
```

Comment:
```suggestion
description: Learn how to implement the append on send feature in your Outlook add-in.
```
Hi, see my comments in Teams but I think we should be consistent here. We've already published "append on send" without hyphens. Suggest you remove throughout the doc, or fix all in the docset. Also, consider referring to as AppendOnSend?